test/services/upload_repository.js

```diff
@@ -15,9 +15,10 @@ import sinonChai from 'sinon-chai';
 chai.use(sinonChai);
 const {expect} = chai;
 
-describe('Upload repository', () => {
+describe('Sheltering repository', () => {
```

Comment:
Top level describe should match the class under test. From file name this would be upload repository?

Reply:
👍